applications/maplab-server-node/test/test-maplab-server-node.cc

```diff
@@ -23,8 +23,14 @@ class MaplabServerNodeTest : public ::testing::Test {
     FLAGS_overwrite = true;
     FLAGS_ros_free = true;
   }
+  uint32_t getTotalNumMergedSubmaps() {
```

Comment:
this is added such that the maplab server node test can access the information if all submaps are merged